repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/tabs.ftl

```diff
@@ -0,0 +1,25 @@
+<#assign hst=JspTaglibs["http://www.hippoecm.org/jsp/hst/core"]>
+
+<#macro tabs tabs>
+    <#if tabs??>
+        <#if tabs.tabsContentBlock.tabPanel?size gt 1>
+            <#assign randPref = tabs.tabsContentBlock.randomPrefix>
+            <div class="nhsuk-tabs" data-module="nhsuk-tabs">
+                <h2 class="nhsuk-tabs__title">${tabs.tabsContentBlock.title}</h2>
+                <div class="nhsuk-tabs__list" role="tablist" aria-label="${tabs.tabsContentBlock.title}" data-module="nhsuk-tabs">
+                    <#list tabs.tabsContentBlock.tabPanel as tabPanel>
+                        <button class="nhsuk-tabs__list-item${(tabPanel?is_first?then(' nhsuk-tabs__list-item--selected',''))}" role="tab" aria-selected="true"

```

Comment:
The cursor doesn't changes as pointer when hovering over the tab titles. Could you take a look ?

Reply:
Good spot.. added prototype PR https://github.com/Manifesto-Digital/hee-prototypes/pull/38

Reply:
Updated css on platform also.